hazelcast/src/main/java/com/hazelcast/map/impl/PartitionContainer.java

```diff
@@ -254,8 +251,8 @@ public void setLastCleanupTimeCopy(long lastCleanupTimeCopy) {
     // There are cases where a global index is used. In this case, the global-index is stored in the MapContainer.
     // By using this method in the context of global index an exception will be thrown.
     // -------------------------------------------------------------------------------------------------------------
-    Indexes getIndexes(String name) {
 
```

Comment:
Minor: maybe we can avoid these additional newlines

Reply:
fixed